private-sessions/src/gramine_ratls/attest.py

```diff
@@ -0,0 +1,190 @@
+"""
+The content of this file is largely inspired from the
+corresponding C implementation of the Gramine gramine-ratls
+utility tool.
+
+TODO: The license information needs to be sorted out
+"""
+
+import ctypes
+
+
+def get_pem_bytes_from_der(
+    header, footer, ctypes_der_ptr, ctypes_der_size
+) -> bytes:
+    """
+    This function loads the mbetls library that comes with gramine
+    installs (mbedtls_gramine.so) and calls the mbedtls_pem_write_buffer
+    to transform certificate in "der" format to one in "pem" format.
+
+    The result is returned as a python byte object and can be directly
+    written to disk.
+
+    This function is largely inspired from the "der_to_pem" function
+    defined in the Gramine implementation of the gramine-ratls utility.
```

Comment:
could you please make more clear what it does instead of how it does (can keep the how but the what should be prio)